pipeline/scripts/cron/5/1/stage-1/test-rados-scrub-scenarios.sh

```diff
@@ -0,0 +1,48 @@
+#! /bin/sh
```

Comment:
Can we have this also included in stage-0 as we have three slots available?

Reply:
added to stage-1.